.eslintrc.js

```diff
@@ -1,10 +1,34 @@
 module.exports = {
+  root: true,
   extends: '@mate-academy/eslint-config',
+  plugins: ['react', 'jest'],
   env: {
-    jest: true
+    es2021: true,
   },
   rules: {
-    'no-proto': 0
+    'no-proto': 0,
   },
-  plugins: ['jest']
+  overrides: [
+    {
+      files: ['src/client/**/*.{js,jsx}'],
+      env: {
+        browser: true,
+      },
+      rules: {
+        'react/jsx-uses-vars': 'error',
+      },
+    },
+    {
+      files: ['src/server/**/*.{js}'],
```

Comment:
Consider adding specific rules for the server-side code if there are any particular coding standards or practices you want to enforce for Node.js environments.